script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_4_3.txt

```diff
@@ -69,24 +69,24 @@ I just need to find some other topic... Ah, that's right.
 <Page20>
 "I was wondering how Akiha is doing. Her school is a boarding school, right?"
 <Page21>
-"It is, indeed.
+"It is indeed.
 It is true that the school Akiha-sama attends is a boarding school. However, as an exception, she is allowed to go to school from home."
 "...#You mean she goes to school from this house?"
 <Page22>
 "Yes, but she is rarely able to make it home before evening like today. After a full day of schoolwork, she has lessons to attend, so it is usually at least seven o'clock by the time she returns to the mansion."
 "Lessons to attend? Such as...?"
 <Page23>
 "Today, she attended violin lessons along with foreign language studies."
-Today? ...Is she on such a schedule every day? She's been forced to practice a lot by the old man for a long time now, so it's probably not a surprise but...
+Today? ...Is she on such a schedule every day? She's been forced to practice a lot by the old man for a long time now, so it's probably not a surprise, but...
 <Page24>
 "In addition, until bedtime, she has to approve the Tohno Group's business report for the day. After waking up and until breakfast time, she goes over the details of future business activities."
 <Page25>
 "―――Seriously?"
 It really is amazing.
-Not only her mind and body but also her lifestyle is made of steel.
+Not only her mind and body, but her lifestyle is also made of steel. //I can't figure out the best place to put "also" here, nothing feels quite right.
```

Comment:
Not just her mind and body, but even her lifestyle is made of steel. ?
> 心身だけでなく、生活習慣まで鋼でできてるのかアイツは。

Reply:
That's pretty good actually